src/project/Doctor.js

```diff
@@ -41,8 +41,19 @@ function _isNpmVersionWithinRanges(npmVersion, ranges) {
 async function _checkNpmVersionAsync(projectRoot) {
   try {
     await Binaries.sourceBashLoginScriptsAsync();
+
+    try {
+      let yarnVersionResponse = await spawnAsync('yarn', ['--version']);
+      let yarnVersion = _.trim(yarnVersionResponse.stdout);
```

Comment:
We're using modern-ish Node everywhere, so I think this should probably use [`String.prototype.trim()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim).